packages/python/plotly/plotly/data/__init__.py

```diff
@@ -102,6 +102,76 @@ def carshare():
     return _get_dataset("carshare")
 
 
+def stocks(indexed=False):
+    """
+Each row in this wide dataset represents closing prices from 6 tech stocks in 2018/2019.
+
+Returns:
+    A `pandas.DataFrame` with 100 rows and the following columns:
+    `['date', 'GOOG', 'AAPL', 'AMZN', 'FB', 'NFLX', 'MSFT']`.
+    If `indexed` is True, the 'date' column is used as the index and the column index
+    is named 'company'
+"""
+    df = _get_dataset("stocks")
+    if indexed:
+        df = df.set_index("date")
+        df.columns.name = "company"
+    return df
+
+
+def experiment(indexed=False):
+    """
+Each row in this wide dataset represents the results of 100 simulated participants
+on three hypothetical experiments, along with their gender and control/treatment group.
+
+
+Returns:
+    A `pandas.DataFrame` with 100 rows and the following columns:
+    `['experiment_1', 'experiment_2', 'experiment_3', 'gender', 'group']`.
+    If `indexed` is True, the data frame index is named "participant"
+"""
+    df = _get_dataset("experiment")
+    if indexed:
+        df.index.name = "participant"
+    return df
+
+
+def short_track_wide(indexed=False):
```

Comment:
How about changing `short_track` to `skating`? Shorter, and easier to understand when you don't know much about the different flavours of skating.

Reply:
Good feedback, thanks! renamed to `medals`